api/MediaDevices.json

```diff
@@ -367,36 +367,22 @@
           "support": {
             "chrome": [
               {
-                "version_added": "52"
+                "version_added": "53"
               },
               {
-                "version_added": "47",
-                "version_removed": "52",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "Experimental Web Platform features",
-                    "value_to_set": "Enabled"
-                  }
-                ],
-                "notes": "Older versions of Chrome implement <code>navigator.webkitGetUserMedia</code>, a prefixed form of the legacy <a href='https://developer.mozilla.org/docs/Web/API/Navigator/getUserMedia'><code>navigator.getUserMedia</code></a> API."
+                "version_added": "52",
+                "version_removed": "53",
+                "prefix": "webkit"
```

Comment:
Hmm, so you're saying there was `navigator.mediaDevices.webkitGetUserMedia` for one release? I tested https://software.hixie.ch/utilities/js/live-dom-viewer/?saved=8570 in Chrome 52 and couldn't confirm. The prefixed variant was `navigator.webkitGetUserMedia`, going back to much further in time.